Formula/quantum++.rb

```diff
@@ -0,0 +1,44 @@
+class Quantumxx < Formula
+  desc "Modern C++ quantum computing library"
+  homepage "https://github.com/softwareQinc/qpp"
+  url "https://github.com/softwareQinc/qpp/archive/v4.1.tar.gz"
+  sha256 "b2d0f84a4726be71e45dc03ad0556ff22f7e6b92c19676391899e1df9091c061"
+  license "MIT"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```

Comment:
Do you need this? i.e. does `brew livecheck quantum++` return wrong results without this?

Reply:
No, will remove and then squash